src/style.css

```
@@ -1,3 +1,163 @@
+* {
+  box-sizing: border-box;
+}
+
+h2 {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  outline: 0;
+  vertical-align: baseline;
+  background: transparent;
+  font-size: 100%;
+}
+
+p {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  outline: 0;
+  vertical-align: baseline;
+  background: transparent;
+  font-size: 100%;
+}
+
+img {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  outline: 0;
+  vertical-align: baseline;
+  background: transparent;
+  font-size: 100%;
+}
+
+button {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  outline: 0;
+  vertical-align: baseline;
+  background: transparent;
+  font-size: 100%;
+}
+
+html {
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+:root {
+  --main-accent: #060b35;
+  --blue-accent: #00acdc;
+  --white-accent: #fff;
+  --secondary: #616070;
+}
+
 body {
+  margin: auto;
+}
+
+.card {
+  margin: auto;
+  padding-left: 16px;
+  padding-right: 16px;
+  border-radius: 5px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  background-color: var(--white-accent);
+}
+
+.card__goods {
+  width: 160px;
+  height: 134px;
+  margin: 32px auto 0;
+  display: block;
+}
+
+.card__info {
+  color: var(--main-accent);
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin-top: 40px;
+  width: 166px;
+}
+
+.card__code {
+
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: var(--secondary);
+
+  padding-top: 4px;
+  padding-bottom: 16px;
   margin: 0;
 }
+
+.card-chosen {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+  align-items: flex-end;
+}
+
+.card-reviews {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: var(--main-accent);
+}
+
+.card__buy {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 0 16px;
+  align-items: center;
+}
+
+.card__price {
+  color: var(--secondary);
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+}
+
+.card__sum {
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 18px;
+  color: var(--main-accent);
+}
+
+.card__button {
+  display: flex;
+  color: var(--white-accent);
+  height: 40px;
+  width: 166px;
```

No need hardcode width, you can write `width: 100%`
```suggestion
  width: 100%;
```

---

https://Pavlo-lab-max-Kopalko.github.io/layout_product-cards/
https://Pavlo-lab-max-Kopalko.github.io/layout_product-cards/report/html_report/